common/src/main/java/com/microsoft/identity/common/internal/eststelemetry/PublicApiId.java

```diff
@@ -30,6 +30,7 @@
     //==============================================================================================
     public static final String PCA_ACQUIRE_TOKEN_SILENT_WITH_PARAMETERS = "21";
     public static final String PCA_ACQUIRE_TOKEN_SILENT_ASYNC_WITH_PARAMETERS = "22";
+    public static final String DEVICE_CODE_FLOW_CALLBACK = "30";
```

Comment:
We'll need to double check if we can use this number.....I'll check that and let you know

Reply:
We can't use this....it is already used by iOS and this range is actually for silent flows. The range for Device Code Flow public API is 600-699. The following are already in use:


Java | AcquireToken(DeviceCodeParameters) | 620
-- | -- | --
Python | acquire_token_by_device_flow | 622
.net | AcquireTokenByDeviceCode | 1008
Node | msal-node/acquireTokenByDeviceCode | 671

Not quite sure why .net is using a number outside the range, but I'm thinking we can pick anything between 600-699. May be 650? I'll let you decide what you want 😁

We have a doc (Excel sheet) with all of these public API ids across platforms. If you don't have that and/or would like to see it, DM me and I will send that over to you via Teams.

Reply:
Ok! I'll change it to 650. I'll ping you on teams, too.